calculation.js

```diff
@@ -0,0 +1,7 @@
+const strictequals = (num1, num2) => {
```

Comment:
They aren't always numbers so `num1` isn't a great name. `val1` would be better.